x-pack/legacy/plugins/apm/server/lib/transaction_groups/fetcher.ts

```diff
@@ -63,6 +63,7 @@ export function transactionGroupsFetcher(
         transactions: {
           terms: {
             ...projection.body.aggs.transactions.terms,
```

Comment:
It's impossible to see what's being spreaded in cases like this. What do you think of
```suggestion
            field: projection.body.aggs.transactions.terms.field,
```

Makes it a lot easier to review :)